doxygen/HowToExamples.dox

```diff
@@ -574,32 +574,32 @@ Example of mesh ICP
 import meshlib.mrmeshpy as mrmeshpy
 
 # Load meshes
-meshFloating = mrmeshpy.loadMesh( "meshA.stl" )
-meshFixed = mrmeshpy.loadMesh( "meshB.stl" )
+meshFloating = mrmeshpy.loadMesh("meshA.stl")
+meshFixed = mrmeshpy.loadMesh("meshB.stl")
 
 # Prepare ICP parameters
-diagonal = meshFixed.getBoundingBox().diagonal();
-icpSamplingVoxelSize = diagonal * 0.01 # To sample points from object
-icpParams = mrmeshpy.ICPProperties()
-icpParams.distTresholdSq = ( diagonal * 0.1 ) ** 2 # Select points pairs that's not too far
-icpParams.exitVal = diagonal * 0.003 # Stop when this distance reached
+diagonal = meshFixed.getBoundingBox().diagonal()
+icp_sampling_voxel_size = diagonal * 0.01  # To sample points from object
+icp_params = mrmeshpy.ICPProperties()
+icp_params.distTresholdSq = (diagonal * 0.1) ** 2  # Select points pairs that is not too far
```

Comment:
`is` => `are`, my bad.
Probably something else like `Limit distance for points pairs used`
Also, there is the same comment in C++ example.